.openshift-ci/pre_tests.py

```diff
@@ -40,6 +40,13 @@ def run(self):
             check=True,
             timeout=PreSystemTests.START_PREFETCH_TIMEOUT,
         )
+        subprocess.run(
+            [
+                "scripts/ci/lib.sh", "image_prefetcher_prebuilt_await"
```

Comment:
@vikin91 I'm not sure this is the best place to do this. Note the calls to this function in `operator/tests/run.sh` and `qa-tests-backend/scripts/run-part-1.sh` - they are intentionally done as late as possible in order to reduce idle waiting time. Adding this call here, right after `image_prefetcher_prebuilt_start` effectively makes the job wait for the prefetcher while it could be doing other things in the meantime.

Also, I don't think `image_prefetcher_prebuilt_await` is idempotent w.r.t. metrics collection, so I'm afraid this addition might duplicate the metrics on the jobs which run this elsewhere 🤔

Reply:
https://github.com/stackrox/stackrox/pull/17372